lib.php

```diff
@@ -338,6 +338,61 @@ function sortvoting_get_sortvoting($sortvotingid) {
     return false;
 }
 
+/**
+ * Get all the responses for a sortvoting.
+ *
+ * @param stdClass $sortvoting
+ * @param stdClass $cm
+ * @param int $groupmode
```

Comment:
It would be clearer if you derived `$groupmode` in this method itself, rather than in calling code:

https://github.com/odeialba/moodle-mod_sortvoting/blob/708a9a0388c2e25bcbf4a40a869c3e321314a0a4/classes/output/sort_voting_results.php#L53

Reply:
In fact you probably don't need it at all - https://github.com/odeialba/moodle-mod_sortvoting/pull/17/files#r1234006487